index.bs

```diff
@@ -522,6 +522,158 @@ See also:
+should be treated as if it never happened,
+and not "queue" the updates and send them after the document becomes fully activated.
+If it makes sense to send the latest state update (and only the latest state,
+not a queue of state changes) to the document after it becomes "[=Document/fully active=]" again,
+consider the "listen for changes" pattern below.
+<div class="example">
+  APIs that periodically send information updates,
+  such as Geolocation API's "[watch position](https://w3c.github.io/geolocation-api/#watchposition-method)"
+  should not send updates if the document is no longer fully active.
+  It also should not queue those updates to arrive later,
+  and only resume sending updates when the document becomes active with the latest location.
+</div>
+
+<h4 id="listen-fully-active">Listen for changes to "[=Document/fully active=]" status</h4>
+
+When a document goes from "[=Document/fully active=]" to non-"[=Document/fully active=]",
+it should be treated similarly to the way discarded documents are treated.
+The document must not retain exclusive access to shared resources
+and must ensure that no new requests are issued
+and that connections that allow for new incoming requests are terminated. 
+When a document goes from non-"[=Document/fully active=]" to "[=Document/fully active=]" again,
+it can restore connections if appropriate.
+  
+While web authors can manually do cleanup (e.g. release the resources, sever connections)
+from within the "pagehide" event and restore them from the "pageshow" event themselves,
+doing this automatically from the API design would reduce a lot of overhead.
+
+<div class="example">
+  APIs that create live connections can pause/close the connection and possibly resume/reopen it later.
+  It's also possible to let the connection stay open to complete existing ongoing requests,
+  and later update the document with the result when it gets restored, if appropriate (e.g.
+  resource loads).
+</div>
+<div class="example">
+  APIs that hold non-exclusive resources
+  may be able release the resource when the document becomes not fully active,
+  and re-acquire them when it becomes "[=Document/fully active=]" again (Screen Wake Lock API is already [doing](https://w3c.github.io/screen-wake-lock/#handling-document-loss-of-full-activity) the first part).
+</div>
+Note: this might not be appropriate for all types of resources,
+e.g. if an exclusive lock is held,
+we cannot just release it and reacquire when "[=Document/fully active=]"
+since another page could then take that lock.
+If there is an API to signal to the page that this has happened,
+it may be acceptable but beware that if the only time this happens is with BFCache,
+then it's likely many pages are not prepared for it. If it is not possible to support BFCache,
+follow the "Discard non-"[=Document/fully active=]" documents for situations that can't be supported" pattern described below.
```

Comment:
You could use the `[[#discard]]` syntax to get a nice autolink.

Reply:
Oooh, that's neat, thanks!